NEWS

```diff
@@ -1,3 +1,49 @@
+etcd v3.2.0 (TODO)
+- faster serializable reads
+- embedded etcd
+  - Etcd.Peers field is now []*peerListener
+  - add Election service
+- native client etcdserver/api/v3client
+  - client "embedded" in the server
+- v3 client
+  - LeaseTimeToLive returns TTL=-1 resp on lease not found
+  - clientv3.NewFromConfigFile is moved to clientv3/yaml.NewConfig
+  - add namespace feature
+  - add Election RPC service to concurrency
+  - let client dial endpoints not in the balancer
+  - add ErrOldCluster with server version checking
+  - translate WithPrefix() into WithFromKey() for empty key
+- v3 etcdctl
+  - add check perf command
+  - add --from-key flag to role grant-permission command
+  - lock command takes an optional command to execute
+- etcd flags
+  - add --enable-v2 flag to configure v2 backend (enabled by default)
+  - add --auth-token flag
+- gRPC proxy
+  - support SRV discovery
+  - coalesce lease requests
+- gateway
+  - support DNS SRV priority
+- auth
+  - support Watch API
+- add server-side logging for large snapshot operations
+- security
+  - deny incoming peer certs with wrong IP SAN
+  - resolve TLS DNSNames when SAN checking
+  - reload TLS certificates for every client requests
+- release
+  - annotate acbuild with supports-systemd-notify
+  - add nsswitch.conf to Docker container image
+  - add experimental ppc64le,arm64 builds
```

Comment:
ppc64le is official / stable since it jumped through all the CI hoops (https://github.com/coreos/etcd/blob/master/Documentation/op-guide/supported-platform.md)